configs/res2net/cascade_mask_rcnn_r2_101_fpn_20e_coco.py

```diff
@@ -0,0 +1,5 @@
+_base_ = '../cascade_rcnn/cascade_mask_rcnn_r50_fpn_20e_coco.py'
+model = dict(
+    pretrained=  # NOQA
+    'https://shanghuagao.oss-cn-beijing.aliyuncs.com/res2net/res2net101_v1b_26w_4s_mmdetv2-f0a600f9.pth',  # NOQA
```

Comment:
You may put 'open-mmlab://res2net101_v1d_26w_4s' here. 
MMCV will be updated later.

Reply:
Configs are updated now.